### DISCUSSION TITLE
Dynamic naming queues

### DISCUSSION THREAD

**Discussion by nullndr** [sidekiq/sidekiq#5578] [Q&A] [answered]
I'm new to ruby and sidekiq, is there an easy way to create a queue dynamically when I run a job?

```ruby
MyJob.set(queue: "My queue").perform_async "data"
```

Will works only if I define `"My queue"` in the `sidekiq.yml` file or if I run `bundle ... -q "My queue"`.

I have a lot of shops that send a lot of webhooks, I would like to assign for each one a queue.

**Comment by mperham** [comment, marked as answer]
Not recommended. Sidekiq isn't designed to handle lots of queues. Just throw them all in `webhooks`.
